install/nano/requirements.txt

```diff
@@ -0,0 +1,28 @@
+pip
+testresources
+setuptools
+futures==3.1.1
+protobuf==3.12.2
+pybind11==2.5.0
+cython==0.29.21
+numpy==1.19.0
+future==0.18.2
+mock==4.0.2
+h5py==2.10.0
+keras_preprocessing==1.1.2
+keras_applications==1.0.8
+gast==0.3.3
+grpcio==1.30.0
+absl-py==0.9.0
+py-cpuinfo==7.0.0
+psutil==5.7.2
+portpicker==1.3.1
+six
+requests==2.24.0
+astor==0.8.1
+termcolor==1.1.0
+wrapt==1.12.1
+google-pasta==0.2.0
+scipy==1.4.1
+pandas==1.0.5
```

Comment:
I think on the other platforms we don't specify the versions except where we know it is need (e.g. h5py), definitely in the conda and pip installations. @tikurahul - what do you think?

Reply:
Yeah. Let's not pin the versions.

Reply:
Versions are no longer pinned